File: hackerrank/week1/javascript/timeconversion.js
function timeConversion(s) {
    // Write your code here
          let newTime = ''
    if (s[8] === 'A') {
        if (s.slice(0, 2) === '12') {
          newTime += '00'
        newTime += s.slice(2, 8)
      } else {
          newTime += s.slice(0, 8)
      }
    } else {
        if (s.slice(0, 2) === '12') {
        newTime += s.slice(0, 8)
      } else {
        let hour = parseInt(s.slice(0, 2)) + 12
        newTime += hour.toString()
        newTime += s.slice(2, 8)
      }
    }
        return newTime

}

File: hackerrank/python/week1/timeconversion.py
def timeConversion(s):
    # Write your code here
    if s[8] == "A":
        if s[0:2] == "12":
            return f'00{s[2:8]}'
        return s[0:8]
    if s[8] == "P":
        if s[0:2] == "12":
            return s[0:8]
        return f'{int(s[0:2]) + 12}' + s[2:8]
